services/travis/settings.py

```diff
@@ -18,4 +19,7 @@
 HOST = 'perma.cc'
 
 # Where we store our generated assets (phantomjs images)
-MEDIA_ROOT = '/tmp/perma/assets'
\ No newline at end of file
+MEDIA_ROOT = '/tmp/perma/assets'
+
+SAUCE_USERNAME = os.environ.get('SAUCE_USERNAME')
```

Comment:
As above, we can just use DJANGO__SAUCE_USERNAME without special handling for these.